AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -1,4 +1,5 @@
 # options
+--swiftversion 5.6
```

Comment:
Nick pointed out that `--swiftversion 999` should work as "always use the latest swift version". Thoughts?